.github/workflows/go-test.yaml

```diff
@@ -15,5 +15,5 @@ jobs:
         uses: actions/setup-go@v2
         with:
           go-version: 1.18
-      - name: test
+      - name: Test
```

Comment:
You can skip name completely, it is displayed only in the workflow logs anyways. What I would change though is the `build:` to `test:` as job name is being displayed directly in the PRs.